src/Tools/SnAdminRuntime/SnAdminRuntime.cs

```diff
@@ -75,11 +75,27 @@ internal static int Main(string[] args)
                 return -1;
 
             Logger.PackageName = Path.GetFileName(packagePath);
-
-            Logger.Create(logLevel, logFilePath);
-            Debug.WriteLine("##> " + Logger.Level);
-
-            return ExecutePhase(packagePath, targetDirectory, phase, parameters, logFilePath, help, schema);
+            try
+            {
+                Logger.Create(logLevel, logFilePath);
+                Debug.WriteLine("##> " + Logger.Level);
+                return ExecutePhase(packagePath, targetDirectory, phase, parameters, logFilePath, help, schema);
+            }
+            catch (System.Reflection.ReflectionTypeLoadException rtlex)
+            {
+                List<string> types = new List<string>();
+                foreach (var item in rtlex.LoaderExceptions) //LoaderExceptions is null? 

```

Comment:
Please check for null according to your comments and remove those comments after that :).